temporalcli/commandsmd/commands.md

```diff
@@ -4,173 +4,390 @@ Commands for the Temporal CLI.
-* `--no-json-shorthand-payloads` (bool) - Always show all payloads as raw payloads even if they are JSON.
+* `--color` (string-enum) - Output coloring. Options: always, never, auto. Default: auto.
+* `--no-json-shorthand-payloads` (bool) - Raw payload output, even if they are JSON.
+
+### temporal activity: Complete or fail an Activity
+
+Update an Activity to report that it has completed or failed. This process
+marks an activity as successfully finished or as having encountered an error
+during execution.
 
-### temporal activity: Complete or fail an Activity.
+Read more: https://docs.temporal.io/cli/activity
 
 #### Options
 
 Includes options set for [client](#options-set-for-client).
 
+### temporal activity complete: Complete an Activity
 
-### temporal activity complete: Complete an Activity.
+Complete an Activity, marking it as successfully finished. Specify 
+the ID and include a JSON result to use for the returned value.
 
-Complete an Activity.
+```
+temporal activity complete \
+    --activity-id=YourActivityId \
+    --workflow-id=YourWorkflowId \
+    --result='{"YourResultKey": "YourResultVal"}'
+```
 
-`temporal activity complete --activity-id=MyActivityId --workflow-id=MyWorkflowId --result='{"MyResultKey": "MyResultVal"}'`
+Read more: https://docs.temporal.io/cli/activity#complete
 
 #### Options
 
-* `--activity-id` (string) - The Activity to be completed. Required.
-* `--identity` (string) - Identity of user submitting this request.
-* `--result` (string) - The result with which to complete the Activity (JSON). Required.
+* `--activity-id` (string) - Activity `ID` to complete. Required.
+* `--result` (string) - Result JSON for completing the Activity. Required.
+* `--identity` (string) - Identity of the user submitting this request
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-### temporal activity fail: Fail an Activity.
+### temporal activity fail: Fail an Activity
+
+Fail an Activity, marking it as having encountered an error during execution.
+Specify the Activity and Workflow Ids.
 
-Fail an Activity.
+```
+temporal activity fail \
+    --activity-id=YourActivityId \
+    --workflow-id=YourWorkflowId
+```
 
-`temporal activity fail --activity-id=MyActivityId --workflow-id=MyWorkflowId`
+Read more: https://docs.temporal.io/cli/activity#fail
 
 #### Options
 
-* `--activity-id` (string) - The Activity to be failed. Required.
-* `--detail` (string) - JSON data describing reason for failing the Activity.
-* `--identity` (string) - Identity of user submitting this request.
-* `--reason` (string) - Reason for failing the Activity.
+* `--activity-id` (string) - `ID` of the Activity to be failed. Required.
+* `--detail` (string) - JSON data describing the reason for failing the Activity
+* `--identity` (string) - Identity of the user submitting this request
+* `--reason` (string) - Reason for failing the Activity
```

Comment:
My textual inconsistency beeper is sounding off wildly. I think it would be better to only use periods for options to separate descriptions and extra attributes. Let's discuss. Not resolving.